includes/widgets/icon-list.php

```diff
@@ -522,7 +522,7 @@ protected function register_controls() {
 			Group_Control_Typography::get_type(),
 			[
 				'name' => 'icon_typography',
-				'selector' => '{{WRAPPER}} .elementor-icon-list-item, {{WRAPPER}} .elementor-icon-list-item a',
+				'selector' => '{{WRAPPER}} .elementor-icon-list-item > .elementor-icon-list-text, {{WRAPPER}} .elementor-icon-list-item > a',
```

Comment:
The list item font-size is still affected when creating a new widget and setting a global link font-size.

Reply:
Fixed

Reply:
The bug still happens - STR:
1. Set a global link font-size.
2. Drag a new icon-list widget.
3. Add a link url to the first list item.
4. The first item's font-size will be larger than the rest.

Reply:
It works good for me.
Are you sure your grunt was activated?
Can you try again and if it's not working for you let's do it together.

Reply:
Confirmed.